roles/telemetry_autoscaling/tasks/test_services.yml

```diff
@@ -17,14 +17,27 @@
   failed_when:
     - result.stdout != "True"
 
-# - name: Verify that custom resource AUTOSCALING is ready
-# It is returning an unexpected response and needs further debugging. will raise another PR to inspect this
-#   ansible.builtin.command:
-#     cmd: |
-#       oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
-#   register: result
-#   failed_when:
-#     - result.stdout != "True"
+- name: Check that the autoscaling CR  is ready
+  block:
+    - name: Verify that custom resource AUTOSCALING is ready
+      ansible.builtin.command:
+        cmd: |
+          oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+      register: result
+      failed_when:
+        - result.stdout != "True"
+  rescue:
+    - name: view autoscaling objects
+      ansible.builtin.command:
+        cmd: "{{ item }}"
+      register: result
+      with_items: [oc get autoscaling autoscaling -oyaml, oc get telemetry telemetry -oyaml]
+      failed_when: result.rc >= 1
```

Comment:
There were some instances where the autoscaling result is giving unexpected response (Expected output is True)
but the job is completing successfully . Hence we added rescue block so that it gives the detailed output and also the job proceeds as expected without failing.

Reply:
Done